asset/js/mapping-block.js

```diff
@@ -43,7 +43,7 @@ mappingMaps.each(function() {
         'Satellite': L.tileLayer.provider('Esri.WorldImagery'),
         'Terrain': L.tileLayer.provider('Esri.WorldShadedRelief')
     };
-    var noOverlayLayer = L.tileLayer.canvas();
+    var noOverlayLayer = new L.GridLayer();
```

Comment:
What's the rationale for switching from `L.tileLayer.canvas()` to `new L.GridLayer()`?

Reply:
It was remove some years ago: https://github.com/Leaflet/Leaflet/commit/64ca1b33b823409901a7b7a55bc6b4bf226f56b4